.github/scripts/wepld_s2_checkpoint_ledger_repair_governance_v43_selftest.py

```
@@ -0,0 +1,358 @@
+
+    # Positive oracle: `raw_root` (main, as checked out) still carries the
+    # PRE-transition documentation - PR #263, which lands the reviewed FINAL
+    # bytes, has not merged yet - so `raw_root` is a valid *base* but can
+    # never itself be a valid *candidate* for `docs_transition`. Build a
+    # candidate whose checkpoint/ledger content is genuinely PR #261's
+    # reviewed FINAL bytes (fetched from this repository's own object
+    # database, which already holds them via previously-fetched history) and
+    # confirm `docs_transition` accepts exactly that pairing.
+    import subprocess
+
+    final_checkpoint = subprocess.check_output(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)